### ISSUE TITLE
Unable to run docker system df on M1 mac

### ISSUE THREAD

**Issue by gayanvirajith** [docker/for-mac#5647]
- [x] I have tried with the latest version of Docker Desktop
  - [x] I have tried disabling enabled experimental features
  - [x] I have uploaded Diagnostics
  - Diagnostics ID: A0899C9E-E18B-49B2-862A-A58AA1798AE8/20210502080401

### Expected behavior

m1 mac should display proper information on having `docker system df`

### Actual behavior

When having `docker system df` it gives below output:

```
Error response from daemon: error getting build cache usage: failed to get usage for f2067y7lsyrj7hxa7etlml9ly: stat /var/lib/docker/overlay2/u07jx4mo3sc7ajegump8l34s6: no such file or directory
```

### Information

  - macOS Version: macOs BigSur
  - Intel chip or Apple chip: Apple chip
  - Docker Desktop Version: Docker Desktop 3.3.1 (63152) 

### Steps to reproduce the behavior

  1. Start docker desktop
  2. Go to terminal
  3. Run `docker system df`

**Comment by gwpries** [comment]
We had this issue on 3.3.1 as well, try 3.3.2 as the http/https proxy situation seems to be fixed

**Comment by stephen-turner** [comment]
Believed fixed in 3.3.2, closing.

**Comment by gayanvirajith** [comment]
Confirmed. Working fine on 3.3.3 (64133). Thanks so much
